examples/tf/graphsage/dist_train.py

```diff
@@ -37,9 +37,11 @@
 flags.DEFINE_string("worker_hosts", "", "worker hosts")
 flags.DEFINE_string("tracker", '/mnt/data/nfs/graph-learn/distributed/','tracker dir')
```

Comment:
Maybe we should also modify the default tracker dir.

Reply:
Added description in readme.